src/Cms/PageRules.php

```diff
@@ -204,7 +204,12 @@ public static function changeTemplate(Page $page, string $template): bool
 			]);
 		}
 
-		if (count($page->blueprints()) <= 1) {
+		$blueprints = $page->blueprints();
+
+		if (
+			count($blueprints) <= 1 ||
```

Comment:
LGTM except following not important one:

I've realized that this line already covered by above line that `$page->permissions()->changeTemplate()`
https://github.com/getkirby/kirby/blob/fix/page-change-template/src/Cms/PagePermissions.php#L46-L48

Reply:
There might be different reasons for this. This line has been in the code before and I wouldn't remove it with this PR.

Reply:
OK